Content/ReleaseBriefings/4_6/Section_NewFeatures.tex

```diff
@@ -46,5 +46,67 @@
 
 %==========================================================================
 
+\begin{frame}[fragile]\label{sec:new_features}
+
+  {\Huge Kokkos::Graph features}
+
+  \vspace{10pt}
+
+\end{frame}
+
+\begin{frame}[fragile]{Kokkos::Graph recap}
+ \begin{itemize}
+     \item Allows creating a graph of kernels
+     \item Executed in order when calling \texttt{submit(ExecutionSpace)}
+      \begin{code}[keywords={auto}]
+    ...
+    auto graph = Kokkos::Experimental::create_graph(exec, [&](auto root) {
+      auto node_A = root.then_parallel_for(
+                    policy_t(exec, 0, 1),
+                    FetchValuesAndContribute(data, index_A, value_A));
+      auto node_B = root.then_parallel_for(
+                    policy_t(exec, 0, 1),
+                    FetchValuesAndContribute(data, index_B, value_B));}
+    ...
+    graph.submit(exec);
+     \end{code}
+ \end{itemize}
+\end{frame}
+
+\begin{frame}[fragile]{Kokkos::Graph new features}
+ \begin{itemize}
+   \item \texttt{then}-node: adds callable to be executed once
+   \item Executed in the \texttt{ExecutionSpace} the graph is submitted to
+     \begin{code}[keywords={auto}]
+    ...
+    auto graph = Kokkos::Experimental::create_graph(exec, [&](auto root) {
+      auto node_A = root.then_parallel_for(
+                    policy_t(exec, 0, 1),
+                    FetchValuesAndContribute(data, index_A, value_A));
+      auto node_B = node_A.then(
+                    FetchValuesAndContribute(data, index_B, value_B));}
+    ...
+    graph.submit(exec);
+     \end{code}
```

Comment:
I thought this was part of the `4.5` release ? https://github.com/kokkos/kokkos-tutorials/blob/21c0ccb69716deb7acabc052857089071116df43/Content/ReleaseBriefings/4_5/Section_GraphsEnhancements.tex#L75

Reply:
yes, I reiterated it here so there is no confusion like "oh, the then nodes are executed on the host?"